.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.3
```

Comment:
This is probably just my ignorance: but what does this mean if you are including that gem in an app that runs for example ruby 2.4?

Reply:
This file is just for a gem development. So all gem tests are also running with this version. It does not force Ruby version which app may use.